src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2386,12 +2408,10 @@ because xbuild doesn't support framework reference assemblies.
     CreatePackagePerAbi="$(AndroidCreatePackagePerAbi)"
     UseSharedRuntime="$(AndroidUseSharedRuntime)"
     Debug="$(AndroidIncludeDebugSymbols)"
-    PreferNativeLibrariesWithDebugSymbols="$(AndroidPreferNativeLibrariesWithDebugSymbols)"
```

Comment:
@pjcollins I believe while this is not used by users, it might have been used by @grendello to help with debugging our runtime in an embedded assemblies scenario. @grendello ever used this property?

Reply:
The scenario for local debugging could still be supported by these changes, you'd just need to change these file names in your common targets: https://github.com/xamarin/xamarin-android/pull/4544/files/319ae454657c479f57c1d0e4e5c6cddfd2ad438d#diff-db39bf07b89b50eaab8ab2001546310cR2196